src/vizy/gclouddialog.py

```diff
@@ -161,6 +163,20 @@ def func():
                 result += self.error_text.out_value(f"An error occurred: {e}") + self.error_dialog.out_open(True)
             return result
 
+        @self.test_sheet.callback()
+        def func():
+            self.kapp.push_mods(self.test_sheet.out_spinner_disp(True), callback_context.client)                        
+            gpsm = self.gcloud.get_interface("KtabularClient")
+            result = self.test_sheet.out_spinner_disp(False)
+            try:
+                data = util.testing.makeDataFrame() # creates a data frame with random values for testing
+                gpsm.createGS('vizy test sheet',data)
```

Comment:
Put a try: except around the creation of the sheet, then return the exception in the message.  See test_image.callback()